src/components/AreaDropdown/AreaDropdown.jsx

```diff
@@ -5,24 +5,28 @@ import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import SchoolIcon from '../../assets/icons/school.svg';
 import TeacherIcon from '../../assets/icons/Teacher.svg';
+import EditAreaModal from '../EditAreaModal/EditAreaModal';
 
-function AreaDropdown({ areaId, areaName, areaStats, areaSites }) {
+function AreaDropdown({ areaId, areaActive, areaName, areaStats, areaSites }) {
   const [isOpen, setIsOpen] = useState(false);
+  const [editAreaModalIsOpen, setEditAreaModalIsOpen] = useState(false);
 
   const toggleDropdown = () => {
     setIsOpen(!isOpen);
   };
 
   return (
-    <div
-      role="button"
-      tabIndex={0}
-      className="area-dropdown"
-      onClick={toggleDropdown}
```

Comment:
Moved `toggleDropdown` `onClick` inside, so that it wouldn't get triggered when someone clicks the edit button.